hypha/apply/determinations/templates/determinations/determination_detail.html

```diff
@@ -4,42 +4,43 @@
 {% block title %}{% trans "Determination for" %} {{ determination.submission.title }}{% endblock %}
 
 {% block content %}
-<div class="admin-bar">
-    <div class="admin-bar__inner">
-        <a class="simplified__submissions-link" href="{{ determination.submission.get_absolute_url }}">
-            {% trans "Back to submission" %}
-        </a>
-        <h2 class="heading heading--no-margin">{% trans "Determination" %} {% if determination.is_draft %}[{% trans "DRAFT" %}]{% endif %}
-        </h2>
-        <h5>{% trans "For" %}: {{ determination.submission.title }}</h5>
-    </div>
-</div>
+    {% adminbar %}
+        {% slot back_link %}
+            <a class="simplified__submissions-link"
+                href="{{ determination.submission.get_absolute_url }}"
+            >
+                {% trans "Back to submission" %}
```

Comment:
On the project side we changed the "Back to …" link text to "View …". I think it would be good to make that change on the submission side as well.

Reply:
Agreed, updated.